appservice/src/deploy/deploy.ts

```diff
@@ -37,7 +39,7 @@ export async function deploy(client: SiteClient, fsPath: string, configurationSe
                 () => {
                     // ignore
                 });
-            client.getAppServicePlan().then(
+            aspPromise.then(
```

Comment:
Don't you need to have failure code, or else do an await?

Reply:
Lol you wrote this "// ignore" failure code

We decided to ignore errors because it's just being used for telemetry in this case and we don't want telemetry to block deployment

Reply:
When I use `aspPromise` elsewhere (aka when it's not just being used for telemetry), I use an await

Reply:
Ah, I see it now.